dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -1068,4 +1061,65 @@ public final Response delete(@Context final HttpServletRequest  request,
         }
     }
 
-}
+    /**
+     * Copies container to the specified host
+     *
+     * @param request
+     * @param response
+     * @param id       id identifier to copy.
+     * @return
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @POST
+    @Path("/{id}/_copy")
+    @JSONP
+    @NoCache
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public final Response copy(@Context final HttpServletRequest request,
+            @Context final HttpServletResponse response,
+            @PathParam("id") final String id) throws DotDataException, DotSecurityException {
+
+        final InitDataObject initData = new WebResource.InitBuilder(webResource)
+                .requestAndResponse(request, response).requiredBackendUser(true)
+                .rejectWhenNoUser(true).init();
+        final User user = initData.getUser();
+        final Host host = WebAPILocator.getHostWebAPI().getCurrentHostNoThrow(request);
+        final PageMode pageMode = PageMode.get(request);
+
+        if (!UtilMethods.isSet(id)) {
+
+            Logger.error(this, "The container id is required");
+            return ExceptionMapperUtil.createResponse(null, "The container id is required");
+        }
+
+        if (!UUIDUtil.isUUID(id)) {
+
+            Logger.error(this, "Container 'id' should be a uuid");
+            return ExceptionMapperUtil.createResponse(null, "Container 'id' should be a uuid");
+        }
+
+        final Container sourceContainer = this.getContainerWorking(id, user,
+                WebAPILocator.getHostWebAPI().getHost(request));
+
+        if (null != sourceContainer && InodeUtils.isSet(sourceContainer.getInode())) {
+
+            ActivityLogger.logInfo(this.getClass(), "Copy Container",
+                    "User " + user.getPrimaryKey() + " saved " + sourceContainer.getTitle(),
+                    host.getHostname());
+
+            Container copiedContainer = this.containerAPI.copy(sourceContainer, host, user,
+                    pageMode.respectAnonPerms);
+
+            Logger.debug(this,
+                    () -> "The container: " + sourceContainer.getIdentifier() + " has been copied");
+
+            return Response.ok(new ResponseEntityView(new ContainerView(copiedContainer))).build();
```

Comment:
As part of the effort to doc new things with swagger, we create an implementation of the ResponseEntityView
that you can return it (without wrapping it on the Response) or you can wrap into the Response but need to extend the ResponseEntityView and type it for instance:
```
ContainerViewResponseEntityView extends ResponseEntityView<ContainerView>
```

This will give enough information to swagger to create a doc for the response

Reply:
Kindly review the recent commit - thanks